### ISSUE TITLE
hittestresstool configuration

### ISSUE THREAD

**Issue by Derryrover** [nens/hittestresstool-client#39]
For the latest version to be deployed to production, all config records for hittestress (not for other apps!) need to be edited.
https://demo.lizard.net/admin/lizard_nxt/clientconfig/

Failing to do this before the prod deploy may crash all hittestress apps on production!

______________________________________________________________________

The following lines need to be added:

   "differenceTemplateUuid": "02cb4e3d-55de-4c06-9a8c-7f4d63928248",
   "differenceMapStyle": "pet_difference:-5:5",
   
   "originalShadeRasterUuid": "01d306ad-62a5-410d-aa47-37f01583cbce",
   "interactiveShadeTemplateUuid": "66cfcd47-fa0c-4533-88e0-839743cabbb8",

   "originalTreesRasterUuid": "357edce2-b802-46d4-8206-fe1ce9f383eb",
   "interactiveTreesTemplateUuid": "fcefa3c0-b538-4648-97e1-b72135918c3f",

   "originalPavedRasterUuid": "8480a74a-ab21-43bc-a1e0-41d38467bc65",
   "interactivePavedTemplateUuid": "76bfd395-ffb3-420e-aceb-c42d1e7bc581"

__________________________________________________________________________
For demo.lizard.net/hittestresstool I already did this:
{
   "minZoom":15,
   "maxBounds":{
      "ne":{
         "lat":51.611301,
         "lng":4.143262
      },
      "sw":{
         "lat":51.58795,
         "lng":4.061294
      }
   },
   "treesStyle":"Greens:0:2000",
   "templateUuid":"a3600181-0f80-4d16-87f1-ee1d9cea6482",
   "initialBounds":{
      "ne":{
         "lat":51.611301,
         "lng":4.143262
      },
      "sw":{
         "lat":51.58795,
         "lng":4.061294
      }
   },
   "pavementsStyle":"lc-fun-1801c",
   "heatstressStyle":"pet_heatstress:38:52",
   "mapboxAccessToken":"pk.eyJ1IjoibmVsZW5zY2h1dXJtYW5zIiwiYSI6ImhkXzhTdXcifQ.3k2-KAxQdyl5bILh_FioCw",
   "originalTreesLayer":"intern:ahn2:tree",
   "originalPavementsLayer":"intern:nl:cover:fun-1801c",
   "originalHeatstressLayer":"nelen-schuurmans:interactive-heat-stress-model",
   
   "differenceTemplateUuid": "02cb4e3d-55de-4c06-9a8c-7f4d63928248",
   "differenceMapStyle": "pet_difference:-5:5",
   
   "originalShadeRasterUuid": "01d306ad-62a5-410d-aa47-37f01583cbce",
   "interactiveShadeTemplateUuid": "66cfcd47-fa0c-4533-88e0-839743cabbb8",

   "originalTreesRasterUuid": "357edce2-b802-46d4-8206-fe1ce9f383eb",
   "interactiveTreesTemplateUuid": "fcefa3c0-b538-4648-97e1-b72135918c3f",

   "originalPavedRasterUuid": "8480a74a-ab21-43bc-a1e0-41d38467bc65",
   "interactivePavedTemplateUuid": "76bfd395-ffb3-420e-aceb-c42d1e7bc581"
}

**Comment by Derryrover** [comment]
@removtilburg

Hi Remo,

See description of this ticket. 
We need to update all the config records, but I have no rights to verify if the hittestresstools still work after making the updates.
Do you have the appropriate rights roles for all the individual hittestresstools?
If so, can you pick this up?

Let me know ;)

kr Tom

**Comment by RemovTilburg** [comment]
@Derryrover 

Ha Tom,


Does this needs to be updated before the demo this thursday? I think it's best to update this when everything is ready to be updated for the customers.

**Comment by Derryrover** [comment]
@RemovTilburg 
Not necessary to do it now. Needs to be done before we deploy to production.
If we foresee changes in this config we can postpone this task to avoid double work.
Maybe you can already verify you have the correct right roles to test the hittestresstools for all the customers.

**Comment by RemovTilburg** [comment]
@Derryrover 

Yes, I have all the roles for all current hittestresstools configs. So I can pick this up once we are ready to deploy everything to production.

**Comment by Derryrover** [comment]
@removtilburg

Hi Remo,

Since hittestress is now on production we also need to update all the configurations for each customer.
It seems this is not yet done for example this one:
https://demo.lizard.net/admin/lizard_nxt/clientconfig/7/change/

This one can be used as an example for which new fields are added:
https://demo.lizard.net/admin/lizard_nxt/clientconfig/1/change/

kr Tom

**Comment by Derryrover** [comment]
@RemovTilburg 

Hi Remo,

Since it is now on production I assume this is finished. allright?
I close this issue, please reopen it if something needs to be done here.

kr Tom
